README.md

```diff
@@ -397,22 +397,30 @@ jsdom.env({
 
 ```js
 var jsdom = require("jsdom");
-var window = jsdom.jsdom().defaultView;
 
-jsdom.getVirtualConsole(window).sendTo(console);
+var virtualConsole = jsdom.createVirtualConsole();
+
+virtualConsole.sendTo(console);
+
+var window = jsdom.jsdom(null, {
```

Comment:
Also maybe just do `var document =` and omit `.defaultView`. If we ever do a major API overhaul it will probably become window-centric but for now jsdom's API is fairly document-centric so it makes sense to leave that in the examples.